spec/lib/query/proposal/listing_spec.rb

```diff
@@ -8,28 +10,68 @@
         FactoryGirl.create(:proposal, status: status)
         proposal = FactoryGirl.create(:proposal, requester: user, status: status)
 
-        with_env_var('ADMIN_EMAILS', user.email_address) do
-          listing = Query::Proposal::Listing.new(user, params)
-          expect(listing.send(status).rows).to eq([proposal])
-        end
+        listing = Query::Proposal::Listing.new(user, params)
+        expect(listing.send(status).rows).to eq([proposal])
       end
 
       context "with an arbitrary client" do
         before do
           user.update_attribute(:client_slug, 'ncr')
+          another_user.update_attribute(:client_slug, 'ncr')
+          yet_another_user.update_attribute(:client_slug, 'gsa18f')
         end
 
-        it "only displays that user's Proposals when they are a client admin" do
-          proposal = FactoryGirl.create(:proposal, requester: user, status: status)
+        describe "when user is not an admin of any kind" do
```

Comment:
Oop, @pkarman and I wrote the wrong set of tests here. This is meant to test the proposal listing on the user dash--Aidan's original test was correct. We'll need to move our permissions tests elsewhere and update them.